finalPrep/annotations.js

```diff
@@ -15,15 +20,22 @@ function buildABear(name, age, fur, clothes, specialPower) {
     isCuddly: true,
   };
 
+//return builtBear
+
   return builtBear
 }
+//call the buildABear function twice with proper arguments
 
 buildABear('Fluffy', 4, 'brown', ['pants', 'jorts', 'tanktop'], 'give you nightmares');
 buildABear('Sleepy', 2, 'purple', ['pajamas', 'sleeping cap'], 'sleeping in');
 
 
 
 //FizzBuzz
+
+/*Create a function, fizzBuzz with parameters num1, num2, range
+iterate from 0 to end of range(parameter)
+*/
 function fizzBuzz(num1, num2, range) {
   for (var i = 0; i <= range; i++) {
     if (i % num1 === 0 && i % num2 === 0) {
```

Comment:
What would you annotate this line as? I would encourage you to look up the `%` operator!